.releaserc.cjs

```diff
@@ -16,25 +16,7 @@ $ run yarn run semantic-release -d
 
 */
 
-const commitAnalyzerSetting = [
-  '@semantic-release/commit-analyzer',
-  {
-    preset: 'angular',
-    releaseRules: [
-      {
-        type: 'feat',
-        release: 'minor',
-      },
-      {
-        type: '*',
-        release: 'patch',
-      },
-    ],
-    parserOpts: {
-      noteKeywords: ['BREAKING CHANGE', 'BREAKING CHANGES'],
-    },
-  },
-];
```

Comment:
I don't know why we are providing these rules when they are already defaults. I _think_ this is maybe breaking the breaking change setup?